Lib/xml/sax/handler.py

```diff
@@ -340,3 +340,47 @@ def resolveEntity(self, publicId, systemId):
                   property_xml_string,
                   property_encoding,
                   property_interning_dict]
+
+
+class LexicalHandler:
+    """Optional SAX2 handler for lexical events.
+
+    This handler is used to obtain lexical information about an XML
+    document, that is, information about how the document was encoded
+    (as opposed to what it contains, which is reported to the
+    ContentHandler), such as comments and CDATA marked section
+    boundaries.
+
+    To set the LexicalHandler of an XMLReader, use the setProperty
+    method with the property identifier."""
```

Comment:
I think having the property identifier string in there was needed for the documentation, even if it's not a valid URL.
```suggestion
    method with the property identifier
    'http://xml.org/sax/handlers/LexicalHandler'."""
```

Reply:
You're right. Thank you, @scoder, for looking at this PR.